### PR TITLE
12-create-url-routing-to-different-page-destinations

### PR DESCRIPTION
**Pull request by lukestern**
Routes added to App:
/
/timeline
/:rover
/:rover/:photoId

Placeholders for static components also added: navbar, weatherbar and footer

### CODE REVIEW

File: src/tests/App.test.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+
```
Comment (oskarwilliams) [inline src/tests/App.test.tsx]: no need for the new line here

File: src/tests/App.test.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import App from '../App';
+
+test('renders landing/home page', () => {
+    render(<App />);
```
Comment (oskarwilliams) [inline src/tests/App.test.tsx:5]: maybe a new line after the render

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: src/tests/App.test.tsx (hunk 1 above)
Comment (lukestern) [reply src/tests/App.test.tsx]: done

File: src/tests/App.test.tsx (hunk 2 above)
Comment (lukestern) [reply src/tests/App.test.tsx:5]: done

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**